sycl/test-e2e/ESIMD/lit.local.cfg

```diff
@@ -10,5 +10,9 @@ if 'gpu-intel-gen9' in config.available_features and platform.system() == 'Windo
 # The current DG2 postcommit job only runs L0 anyway,
 # so there's no difference in coverage.
 # We should investigate why OCL fails separately.
-if 'gpu-intel-dg2' in config.available_features:
+
+# GPUs to disable ESIMD tests on.
+gpus_disable_esimd = ['gpu-intel-dg2', 'gpu-intel-pvc']
```

Comment:
comment/varname is not accurate IMO, i would say `gpus_requiring_level_zero` or something